### PR TITLE
docs(all): Spelling

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
There are a few typos in the ristdocs.

# Changes
- Update the spelling dictionary to include terms such as UTXO.
- Fix typos

# Tests
N/A

### CODE REVIEW

**Comment by peterpeterparker** [comment]
> There are a few typos in the ristdocs.

Haha 😁

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: .config/spellcheck.dic [hunk 1]
```diff
@@ -3,3 +3,22 @@ Ethereum
...
+utxos
+UTXOs
+Oisy
+PAPI
+backend
+backend's
+readonly
+pricelist
+ckBTC
++
```
Comment (AntonioVentilii) [inline .config/spellcheck.dic:20]: is this one required? really? eheh! just curiosity

Comment (bitdivine) [reply .config/spellcheck.dic:20]: Yup.  It is possible to specify patterns but the syntax for that is so arcane that it isn't worth it for us.

**Pull request merged**